### PR TITLE
Provide mechanism to disable buildship

### PR DESCRIPTION
**Pull request by snjeza**
Fixes https://github.com/eclipse/eclipse.jdt.ls/issues/257
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/497

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,6 +111,12 @@
           "description": "Traces the communication between VS Code and the Java language server.",
           "scope": "window"
         },
+        "java.skip.gradleImporter": {
```
Comment (fbricon) [inline package.json]: To be consistent with the existing `java.import.exclusions` preference and others toggle like prefs, we should have `java.import.gradle.enabled`

**Comment by snjeza** [comment]
@fbricon I have updated the PR.

File: package.json [hunk 2]
```diff
@@ -111,6 +111,18 @@
...
+        "java.import.gradle.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable the gradle importer.",
+          "scope": "window"
+        },
+        "java.import.maven.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable the maven importer.",
```
Comment (fbricon) [inline package.json]: Maven

File: package.json [hunk 3]
```diff
@@ -111,6 +111,18 @@
           "description": "Traces the communication between VS Code and the Java language server.",
           "scope": "window"
         },
+        "java.import.gradle.enabled": {
+          "type": "boolean",
+          "default": true,
+          "description": "Enable/disable the gradle importer.",
```
Comment (fbricon) [inline package.json]: Gradle

**Comment by snjeza** [comment]
@fbricon I have updated the PR.

**Pull request merged**